wps-cloudformation-template.yaml

```diff
@@ -623,82 +698,50 @@ Resources:
-        "            if env_var.startswith(http_prefix) and not env_var.endswith(filename_suffix) and env_var_value != http_prefix:",
-        "                put(filename, env_var_value)",
-        "",
-        "        return send(event, context, SUCCESS, responseData, 'CustomResourcePhysicalID')",
-        "    except Exception as e:",
-        "        print(e)",
-        "        return send(event, context, FAILED, responseData, 'CustomResourcePhysicalID')",
-        "",
-        "",
-        "def put(filename, url):",
-        "    print('Downloading url : {}'.format(url))",
-        "    request = Request(url)",
-        "    response = urlopen(request)",
-        "",
-        "    file_contents = response.read()",
-        "    s3c.put_object(Bucket=target_bucket, Key='{}{}'.format(config_location, filename), Body=file_contents)",
-        "    print('Downloaded url : {}'.format(url))",
-        "",
-        "",
-        "def send(event, context, responseStatus, responseData, physicalResourceId):",
-        "    responseUrl = event['ResponseURL']",
-        "",
-        "    responseBody = {}",
-        "    responseBody['Status'] = responseStatus",
-        "    responseBody['Reason'] = 'See the details in CloudWatch Log Stream: ' + context.log_stream_name",
-        "    responseBody['PhysicalResourceId'] = physicalResourceId",
-        "    responseBody['StackId'] = event['StackId']",
-        "    responseBody['RequestId'] = event['RequestId']",
-        "    responseBody['LogicalResourceId'] = event['LogicalResourceId']",
-        "    responseBody['Data'] = responseData",
-        "",
-        "    json_responseBody = json.dumps(responseBody)",
-        "",
-        "    print('Response body:' + json_responseBody)",
-        "",
-        "    headers = {",
-        "        'content-type' : '',",
-        "        'content-length' : str(len(json_responseBody))",
-        "    }",
-        "",
-        "    try:",
-        "        response = requests.put(responseUrl,",
-        "                                data=json_responseBody,",
-        "                                headers=headers)",
-        "        print('Status code: ' + response.reason)",
-        "    except Exception as e:",
-        "        print('send(..) failed executing requests.put(..): ' + str(e))",
-        ""
-        ]]
+        ZipFile: |
+          import os
+          import boto3
+          from urllib2 import urlopen, Request
+          import cfnresponse
+
+          responseData = {'Data': 'OK'}
+
+          s3c = boto3.client('s3')
+          target_bucket = os.getenv('BUCKET')
+          config_location = os.getenv('CONFIG_LOCATION')
+          filename_suffix = os.getenv('FILENAME_SUFFIX')
+          http_prefix = os.getenv('HTTP_FILE_PREFIX')
+
+
+          def handler(event, context):
+             try:
+                 if event['RequestType'] == 'Create':
```

Comment:
What if we want to update the config files? Maybe allow the update request type as well